<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketCart</class>
 <widget class="QDialog" name="MarketCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
QCommandLinkButton
{
}
QDialog
{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #28d, stop: 0.1 #39e, stop: 0.49 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width:70px;
min-height: 17px;
max-height: 17px;
}</string>
  </property>
  <widget class="QLabel" name="itemNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemNameTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWebView" name="webView" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>851</width>
     <height>521</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="itemNameErrorLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No Source is selected </string>
   </property>
  </widget>
  <widget class="QComboBox" name="amazonComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="flipKartPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width:70px;
min-height: 17px;
max-height: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/fk.png</normaloff>:/images/fk.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="amazonPushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width:70px;
min-height: 17px;
max-height: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/amazon.jpg</normaloff>:/images/amazon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="homeShop18PushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0, stop: 0.1 #0e, stop: 0 #47c, stop: 0.5 #66b, stop: 1 #77c);
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width:70px;
min-height: 17px;
max-height: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/hs.png</normaloff>:/images/hs.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="flipKartComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="homeShop18ComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="flipKartRadioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>103</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="amazonRadioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>103</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="homeShop18RadioButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>103</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
